stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -15,7 +15,6 @@ export
     axpy!,
     axpby!,
     blascopy!,
-    dot,
```

Comment:
maybe just comment it out, so it's clear that it's no longer exported ? 
`#dot` ?